### DISCUSSION TITLE
Managing Logs

### DISCUSSION THREAD

**Discussion by alejoGT1202** [argoproj/argo-workflows#7778] [Q&A] [answered]
Hello,

I'm a newbie with Argo and orchestration tools and I've notice that there are some orchestration tools that allow you to get an historical of your logs. I was wondering it this is possible with Argo. Since currently I can only see the logs from the pods that are been executed, but not for example the logs for pods that were executed 5 hours ago. Thanks for the help.

**Comment by terrytangyuan** [comment, marked as answer]
You can configure archive logs: https://github.com/argoproj/argo-workflows/blob/master/docs/configure-archive-logs.md
